go.mod

```diff
@@ -3,9 +3,8 @@ module github.com/envoyproxy/gateway
 go 1.19
 
 require (
-	github.com/cncf/xds/go v0.0.0-20220314180256-7f1daf1720fc
+	github.com/cncf/xds/go v0.0.0-20230112175826-46e39c7b9b43
```

Comment:
@jcheld at this point in the release, I prefer not bumping deps. Please revert go.mod/go.sum